docs/source/quickstart.rst

```diff
@@ -84,7 +80,7 @@ To do this, you can either
 
 1. Install Chaco and its :ref:`dependencies` from `PyPI
    <http://pypi.python.org/pypi>`_ using `easy_install
-   <http://packages.python.org/distribute/easy_install.html>`_ (part of
```

Comment:
if you are referring to the link in https://github.com/enthought/chaco/pull/292/files?diff=split#diff-de111c49c3e97d38ba7482e8976fa161R90. Yes I think that this is not valid anymore. But I am not sure what you mean about Canopy.